lib/app_text/second_page_text.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+
+class AppText {
+  static const TextStyle textBold = TextStyle(
+    fontSize: 14,
+    fontWeight: FontWeight.bold,
+  );
+
+  static TextStyle textBoldColor = textBold.copyWith(
+    color: const Color(0xFF008080),
+  );
```

Comment:
textBoldColor nie mający  `fontWeight: FontWeight.bold`? to na pewno ma tak być? I nie rób kolorowych tekstów, żeby był colorek na tekstach używaj `AppText,textBold.copyWith(color: Color(0x.....)` etc

Reply:
w ekranie robisz `Text('tratatat', style: AppText.textBold.copyWith(color: Color(0x......))` a nie w AppText byczq